main/samba/APKBUILD

```diff
@@ -128,6 +130,7 @@ initscript() {
 	depends=""
 	install -Dm755 $srcdir/$pkgname.initd "$subpkgdir"/etc/init.d/$pkgname
 	install -Dm644 $srcdir/$pkgname.confd "$subpkgdir"/etc/conf.d/$pkgname
+	install -dm755 "$subpkgdir"/run/$pkgname
```

Comment:
`/run` is required to be cleared at reboot, so I don't know if it makes sense to create it on package install time, since it needs to be recreated on service start anyway.

https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s15.html

Reply:
So, it must have done in service script. I will revise this PR along with other changes.